validation-test/BuildSystem/skip_clean_corelibs.test

```diff
@@ -0,0 +1,20 @@
+# REQUIRES: standalone_build
+# UNSUPPORTED: OS=macosx
```

Comment:
Do I need to mark all Darwin platforms like iOS as unsupported here? I had a previous BuildSystem test pass the CI, then we had to go back and remove all Darwin platforms later, ie #38279.